examples/ekf_vins_one.py

```diff
@@ -0,0 +1,85 @@
+# %%
+from miluv.data import DataLoader
+import utils.liegroups as liegroups
+import miluv.utils as utils
+import examples.ekfutils.vins_one as vins_one
+import examples.ekfutils.postprocessing as postprocessing
+import examples.ekfutils.common as common
+
+import numpy as np
+import pandas as pd
+
+#################### EXPERIMENT DETAILS ####################
+exp_name = "13"
+
+#################### LOAD SENSOR DATA ####################
+miluv = DataLoader(exp_name, imu = "px4", cam = None, mag = False)
+data = miluv.data["ifo001"]
+vins = utils.load_vins(exp_name, "ifo001", loop = False, postprocessed = True)
+
+#################### ALIGN SENSOR DATA TIMESTAMPS ####################
+query_timestamps = np.append(
+    data["uwb_range"]["timestamp"].to_numpy(), data["height"]["timestamp"].to_numpy()
+)
+query_timestamps = query_timestamps[query_timestamps > vins["timestamp"].iloc[0]]
+query_timestamps = query_timestamps[query_timestamps < vins["timestamp"].iloc[-1]]
+query_timestamps = np.sort(np.unique(query_timestamps))
```

Comment:
maybe state in comments that we want all range measurements between the first and the last recorded vins data

Reply:
I was planning to include this level of detail in the supplementary documentation